src/docfx/Errors.cs

```diff
@@ -127,6 +127,12 @@ public static Error ExceedGitHubRateLimit()
         public static Error GitHubUserNotFound()
             => new Error(ErrorLevel.Warning, "github-user-not-found", $"User not found on GitHub");
 
+        public static Error NewtonJsonSchemaLimitExceeded(string message)
```

Comment:
NewtonsoftJsonSchemaLimitExceeded?

Reply:
+1

---
In reply to: [208163143](https://github.com/dotnet/docfx/pull/3164#discussion_r208163143) [](ancestors = 208163143)